### DISCUSSION TITLE
Started PR to include Zigbee2MQTT into openhab

### DISCUSSION THREAD

**Discussion by Larsen-Locke** [Koenkk/zigbee2mqtt#12207] [General]
Hi Koen,
I like z2m due to it's huge hardware support and transparent functionality. Unfortunately your preferred platform seems to be home assisstant...?
Personally I prefer openhab due to it's great community and openess. Untill now z2m was not included in the openhab setup.
That's why I created a PR to include the setup/update/remove routines into openhabian (https://www.openhab.org/docs/installation/openhabian.html)
The PR is this:
https://github.com/openhab/openhabian/pull/1670
I basically followed your install and update documentation, so it should be fine.

Please let me know if you have further recommendation...
Regards, Carsten

**Comment by Koenkk** [comment]
Looks good! Zigbee2MQTT will work with every system supporting MQTT 😄

**Comment by Larsen-Locke** [comment]
I know (and love it). The manual installation scared off quite a few new users.
That’s why I wanted an installation option in the setup that handles git, install, usb-port etc.
PR was merged today. 
Have a nice day
Carsten

**Reply by Koenkk** [reply]
Might be good to update https://www.zigbee2mqtt.io/guide/usage/integrations.html , could you make a pr for it?

**Reply by Larsen-Locke** [reply]
I made I first start. 
Could you give me a hint which viewer to use to check the result?

**Reply by Koenkk** [reply]
You can use visual studio code to render markdowns.

**Reply by Larsen-Locke** [reply]
Sorry for my bad PR of the documentation. It wasn't finished and you corrected it...
Could you please have a short look at the [configuration.yaml](https://github.com/Larsen-Locke/openhabian/blob/1666_zigbee2mqtt/includes/zigbee2mqtt/configuration.yaml) that I put in the setup routine?
Especially:
- As port I chose 8081 to avoid conflict with default 8080 of openhab. You're fine with that?
- Should I include those legacy parameters?
- Should I include the options to generate pan_id and network_key?
- As output I use attribute_and_json because without json the web-interface does not work properly.
Well, that's it. Any recommendations are welcome.
